File: src/main/java/seremis/geninfusion/soul/gene/newAI/moveIndoors/GeneAIMoveIndoors.java
package seremis.geninfusion.soul.gene.newAI.moveIndoors;

import seremis.geninfusion.api.soul.IAllele;
import seremis.geninfusion.api.soul.lib.Genes;
import seremis.geninfusion.soul.MasterGene;
import seremis.geninfusion.soul.allele.AlleleBoolean;

public class GeneAIMoveIndoors extends MasterGene {

    public GeneAIMoveIndoors() {
        addControlledGene(Genes.GENE_AI_MOVE_INDOORS_INDEX);
    }

    @Override
    public Class<? extends IAllele> possibleAlleles() {
        return AlleleBoolean.class;
    }
}

File: src/main/java/seremis/geninfusion/soul/gene/newAI/GeneAIMoveIndoors.scala
package seremis.geninfusion.soul.gene.newAI

import seremis.geninfusion.api.soul.lib.Genes
import seremis.geninfusion.soul.MasterGene

class GeneAIMoveIndoors extends MasterGene(classOf[Boolean]) {
    addControlledGene(Genes.GENE_AI_MOVE_INDOORS_INDEX)
}
